workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -503,6 +491,27 @@ def _diurnal_func(
             return _assign_diagnostic_time_attrs(diag, prognostic)
 
 
+@add_to_diags("physics")
+@diag_finalizer("histogram")
+@transform.apply("resample_time", "3H", inner_join=True)
+@transform.apply("subset_variables", list(HISTOGRAM_BINS.keys()))
+def compute_histogram(prognostic, verification, grid):
```

Comment:
Can you add a test for this function and consider refactoring it to vcm? histograms are pretty general purpose functionality. BTW there is https://github.com/xgcm/xhistogram.git, but I don't see the need to add the dependency unless we really need to daskify this computation.

Reply:
yes, I saw that package and think @spencerkclark has used it to good effect. but seems like it's under active/early development, so would rather avoid the dep.

I can refactor to VCM—but need to think a bit about the right format for the output. Specifically, it's helpful to know the bin widths. Right now I'm using a separate DataArray for them (so basically the output is a dataset w/ two arrays) but it might be better to encode it in the coordinates somehow.

Reply:
There is effort ongoing to refine and move the functionality of xhistogram to xarray (https://github.com/pydata/xarray/issues/4610, https://github.com/pydata/xarray/pull/5400), so I agree it probably would make sense to wait until that occurs.